### ISSUE TITLE
"Run Test" takes a variable amount of time to run in contrast to "Debug Test"

### ISSUE THREAD

**Issue by 188599** [OmniSharp/omnisharp-vscode#5126]
## Issue Description ##

I've noticed for some time that whenever I use the "Run Test" button it would sometimes take a lot longer than expected to run even the simplest of tests, so much so that I would end preferring to use "Debug Test" over it as it would work in a much more stable fashion.

## Logs ##

Logs from a "Run Test" that took much longer than anticipated

```
[xUnit.net 00:00:00.00] xUnit.net VSTest Adapter v2.4.3+1b45f5407b (64-bit .NET Core 3.1.23)
[xUnit.net 00:00:00.57]   Discovering: Catalogo.Core.Domain.UnitTest
[xUnit.net 00:00:01.77]   Discovered:  Catalogo.Core.Domain.UnitTest
[xUnit.net 00:00:00.00] xUnit.net VSTest Adapter v2.4.3+1b45f5407b (64-bit .NET Core 3.1.23)
[xUnit.net 00:00:00.73]   Starting:    Catalogo.Core.Domain.UnitTest
[xUnit.net 00:00:47.08]   Finished:    Catalogo.Core.Domain.UnitTest
----- Test Execution Summary -----

Catalogo.Core.Domain.UnitTest.Web.CotaRepresentanteDomainWebUnitTest.SillyLittle_Test:
    Outcome: Passed
    
Total tests: 1. Passed: 1. Failed: 0. Skipped: 0
```

Logs from the "Debug Test" run over the same test:

```
[xUnit.net 00:00:00.00] xUnit.net VSTest Adapter v2.4.3+1b45f5407b (64-bit .NET Core 3.1.23)
[xUnit.net 00:00:00.64]   Discovering: Catalogo.Core.Domain.UnitTest
[xUnit.net 00:00:01.87]   Discovered:  Catalogo.Core.Domain.UnitTest
Started debugging process #16180.
[xUnit.net 00:00:00.01] xUnit.net VSTest Adapter v2.4.3+1b45f5407b (64-bit .NET Core 3.1.23)
[xUnit.net 00:00:01.69]   Starting:    Catalogo.Core.Domain.UnitTest
[xUnit.net 00:00:03.56]   Finished:    Catalogo.Core.Domain.UnitTest
Debugging complete.
```

The test itself was just this:

```cs
[Fact]
public void SillyLittle_Test()
{
    Assert.True(true);
}
```

As an additional note, the class this test was run on is using the same fixtures we use to run our tests in a Firebird database, but even so, I can't understand why do the run times vary so greatly and why the same doesn't apply to the debug tests, which is very consistent in regards to time.

### OmniSharp log ###
<details>Post the output from Output-->OmniSharp log here</details>

### C# log ###
<details>Post the output from Output-->C# here</details>

## Environment information ##

**VSCode version**: 1.65.2
**C# Extension**: 1.24.1

<details><summary>Dotnet Information</summary>
.NET SDK (reflecting any global.json):
 Version:   6.0.201
 Commit:    ef40e6aa06

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19044
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\6.0.201\

Host (useful for support):
  Version: 6.0.3
  Commit:  c24d9a9c91

.NET SDKs installed:
  3.1.417 [C:\Program Files\dotnet\sdk]
  6.0.201 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 3.1.23 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 6.0.3 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.1.23 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.3 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.23 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.3 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

To install additional .NET runtimes or SDKs:
  https://aka.ms/dotnet-download
</details>
<details><summary>Visual Studio Code Extensions</summary>

|Extension|Author|Version|
|---|---|---|
|csharp|ms-dotnettools|1.24.1|
|dotnet-core-essentials|KishoreIthadi|0.0.8|
|dotnet-test-explorer|formulahendry|0.7.7|
|github-vscode-theme|GitHub|6.0.0|
|material-icon-theme|PKief|4.14.1|
|ng-template|Angular|13.3.0|
|test-adapter-converter|ms-vscode|0.1.5|
|vscode-eslint|dbaeumer|2.2.2|
|vscode-npm-script|eg2|0.3.24|
|vscode-peacock|johnpapa|4.0.1|
|vscode-test-explorer|hbenl|2.21.1|;

</details>

**Comment by 188599** [comment]
> Are you able to really reproduce this with just that SillyLittle_Test? The tests from the output have different names, so I am just double checking.

Yes, it was that test it was running, I think the output is slightly different because when on the debug mode it doesn't print the tests output, but you can see it's calling the same classes.

Although, like I mentioned before, the test itself is being ran with a few of our fixtures which I can only imagine are the main reason behind this, but I have no clue on how to turn into something you could easily reproduce without a good chunk of our code base.

Is there any sort of log I can enable to give you a better clue around this issue?

**Comment by JoeRobich** [comment]
Closing due to lack activity. If you are still having problems, please open a new issue at https://github.com/omnisharp/omnisharp-roslyn/issues
